addons/mail/static/src/core/common/chat_hub_model.js

```diff
@@ -121,6 +123,8 @@ export class ChatHub extends Record {
         // folded before opened because if there are too many opened they will be added to folded
         this.folded = toFold.filter((chatWindow) => chatWindow.exists());
         this.opened = toOpen.filter((chatWindow) => chatWindow.exists());
+        this.loaded = true;
+        this.loadCbs.forEach((fn) => fn());
```

Comment:
This executes at each load, you might want that in static new

Reply:
oops forgot to clear